### PR TITLE
Add Analytics screenshots

### PR DESCRIPTION
**Pull request by francho**

### CODE REVIEW

File: _includes/common-modal.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+                <img src="{{site.baseurl}}/assets/img/screenshots/{{include.screenshot}}"
+                     alt="frogtek analytics img-fluid">
+                {% endif %}
+                {% if include.content %}
+                {{ content }}
+                {% endif %}
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (cati-risquez) [inline _includes/common-modal.html]: Add line

File: _layouts/analytics.html [hunk 2]
```diff
@@ -36,6 +37,17 @@
...
+    $('.analytics-carousel').owlCarousel({
+      autoplay: true,
+      autoplayHoverPause: true,
+      dots: true,
+      items: 1,
+      loop: true,
+      margin: 20,
+      stagePadding: 0,
+    });
+
```
Comment (cati-risquez) [inline _layouts/analytics.html]: Remove line?

**Review by cati-risquez — APPROVED** [review]

File: _includes/common-modal.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<div class="modal fade" id="{{include.id}}" tabindex="-1" role="dialog">
```
Comment (durey) [inline _includes/common-modal.html:1]: @francho Just try not to omit the required spaces in variable anchors: `{{ include.id }}` (there are a lot of cases like this)

File: _sass/frogtek-theme.scss [hunk 4]
```diff
@@ -377,6 +377,39 @@ footer {
   }
 }
 
+.analytics-carousel {
+  .owl-dots {
+    color: $color-frogtek-orange;
```
Comment (durey) [inline _sass/frogtek-theme.scss:382]: @francho  Including the name of the colour as part of the name of the variable is usually a bad choice (nowadays the semantics point of view is key)

**Pull request merged**